code/obj/machinery/manufacturer.dm

```diff
@@ -1854,12 +1829,12 @@
 
 		return dat.Join()
 
-	proc/eject_manudrive(var/mob/living/user)
+	proc/eject_manudrive(mob/living/user)
 		src.drive_recipes = null
 		user.put_in_hand_or_drop(manudrive)
 		src.manudrive = null
 
-	proc/load_item(var/obj/item/O,var/mob/living/user)
+	proc/load_item(obj/item/O,mob/living/user)
```

Comment:
```suggestion
	proc/load_item(obj/item/O, mob/living/user)
```
Noting down 'add spaces after commas to the style guide'

Reply:
or newlines in the cases of lists @frawhst :)